src/index.html

```diff
@@ -6,9 +6,89 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect"href="https://fonts gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400;500&family=Roboto:wght@500&display=swap"
+      rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home">
+        <img src="./images/logo.png"
+          alt="MOYO logo"
+          class="icon"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav-list">
+          <li class="nav-item">
+            <a href="#apple"
+              class="nav-link is-active"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li class="nav-item">
+            <a href="#samsung"
+              class="nav-link"
+            >
```

Comment:
See how we [move attributes to a new line](https://mateacademy.slack.com/archives/C050SHYFFR9/p1681912970291919):